### ISSUE TITLE
[Bug]: Deepseek vl2 model decoding phase has a very worse performance in both online and offline inference

### ISSUE THREAD

**Issue by beyondHJM** [vllm-project/vllm#25229]
### Your current environment

We use deepseek_vl2_tiny model with only 3B weights, the throughput in decoding phase is less than 100 for a single prompt, and using CUDA graph performs worse than not using CUDA graph.

### 🐛 Describe the bug

from vllm import LLM, EngineArgs, SamplingParams
from PIL import Image
import torch
from dataclasses import asdict
import time
model_name = "deepseek-ai/deepseek-vl2-tiny"

engine_args = EngineArgs(
    model=model_name,
    max_model_len=4096,
    max_num_seqs=1,
    hf_overrides={"architectures": ["DeepseekVLV2ForCausalLM"]},
    limit_mm_per_prompt={"image": 1},
    enforce_eager=True #False
)

engine_args = asdict(engine_args) | {
    "seed": None,
    # "mm_processor_cache_gb": 0 if args.disable_mm_processor_cache else 4,
}
llm = LLM(**engine_args)

data = Image.open("/home/admin/workspace/aop_lab/app_source/q0.jpeg")
img_questions = [
    "提供了商品的基本外观信息. \n<|ref|>山姆超市 海盐苏打饼干1.5kg 休闲零食点心 正品代购<|/ref|>请简要描述一下此商品的相关属性。",
]

prompts = [
    f"<|User|>: <image>\nPlease describe this picture\n\n<|Assistant|>:" 
]
inputs = [
    {
        "prompt": prompts[i % len(prompts)],
        "multi_modal_data": {'image': data},
    }
    for i in range(len(prompts))
]

stop_token_ids = [93532, 93653, 944, 93421, 1019, 93653, 93519]
# stop_token_ids=None
sampling_params = SamplingParams(
    temperature=0.2, max_tokens=256, stop_token_ids=stop_token_ids
)

outputs = llm.generate(
    inputs,
    sampling_params=sampling_params,
)

print("-" * 50)
for o in outputs:
    generated_text = o.outputs[0].text
    print(generated_text)
    print("-" * 50)
exit(0)

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.
